src/apps/media/src/app/components/MediaSelected.js

```diff
@@ -88,9 +89,22 @@ export function MediaSelected(props) {
                             params={"?w=200&h=200&type=fit"}
                           />
                         </figure>
-                        <button className={styles.Check} aria-label="Checked">
-                          <FontAwesomeIcon icon={faCheck} />
-                        </button>
+                        <Button
+                          variant="contained"
+                          size="small"
+                          aria-label="Checked"
+                          sx={{
```

Comment:
Is there a reason we are using inline styles for these?

Reply:
@shrunyan this is the MUI convention and the method we used Marketing website. In order to clean up classname styling I used this method. If we want to move away from this method we can also write it with styled component. 

Side notes * The sx isn't inline styling it does not show the styles in the dom.